WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreViewModel.kt

```diff
@@ -330,11 +331,18 @@ class RestoreViewModel @Inject constructor(
                         if (contentState.type == ViewType.PROGRESS) {
                             contentState as JetpackListItemState.ProgressState
                             contentState.copy(
-                                    progress = state.progress ?: 0,
+                                    progress = restoreStatus.progress ?: 0,
                                     progressLabel = UiStringResWithParams(
-                                            string.restore_progress_label,
-                                            listOf(UiStringText(state.progress?.toString() ?: "0"))
-                                    )
+                                            R.string.restore_progress_label,
+                                            listOf(UiStringText(restoreStatus.progress?.toString() ?: "0"))
+                                    ),
+                                    progressInfoLabel = if (restoreStatus.currentEntry != null) {
+                                            UiStringText("${restoreStatus.currentEntry}")
+                                        } else {
+                                            null
+                                        },
+                                    progressStateLabel = UiStringText("${restoreStatus.message}"),
```

Comment:
⚠️ It seems that the "message" field is also nullable. I think this logic will display "null" to the user, wdyt?

Reply:
Downstream it's `UiHelpers.setTextOrHide` that handles setting the view itself and nulls are properly handled there, so we are good. 👍  I updated to handle null message in commit eb6088e